app/presenters/budget_xml_presenter.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class BudgetXmlPresenter < SimpleDelegator
+  def period_start_date
+    return if super.blank?
+    I18n.l(super, format: :iati)
+  end
+
+  def period_end_date
+    return if super.blank?
+    I18n.l(super, format: :iati)
+  end
+
+  def value
+    super.to_s
+  end
+
+  def budget_type
```

Comment:
The presenter is cool - I like it! 

I did expect to to see code here to translate the type and status into the numeric value, it might be that I'm not loving all the locale stuff but it feels more like the presenters job to transalate the type into its value rather than the locale file, the XML is the only place it needs to be numeric, right?

What do you think about something like:

````ruby
BUDGET_TYPES = {
"original": "1",
"updated": "2"
}

def budget_type
return if super.blank?
BUDGET_TYPES[super]
end

Reply:
Good point, I have made this needlessly complex!! 🤦‍♀